src/bridge/cockpitpeer.c

```diff
@@ -74,6 +74,7 @@ struct _CockpitPeer {
   gboolean closed;
   gchar *problem;
   JsonObject *failure;
+  gboolean poisoned;
```

Comment:
This should get a comment what this means exactly. Perhaps also a more descriptive name? `dropped_privileged`?

Reply:
I kept the name and added comments.